troubleshooting/known_issues/cfs_session_status_for_image_customization_on_remote_node_stuck_at_running.md

```diff
@@ -0,0 +1,103 @@
+# CFS Session for Image Customization on Remote Node Status Stuck at `running`
+
+## Issue description
+
+A CFS session for image customization job on a remote node can get stuck in the `running` state indefinitely.
+This can occur during image customization if any of the following things happen:
+
+- The remote node reboots or crashes.
+- The IMS job container on the remote build node is killed or stopped.
+
```

Comment:
This bug is fixed in CSM x.y.z. Prior to that CSM version, the procedures on this page must be followed to identify and resolve the issue.